libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -116,6 +117,8 @@ + (instancetype)sharedInstance {
     static SFUserAccountManager *userAccountManager = nil;
     dispatch_once(&pred, ^{
 		userAccountManager = [[self alloc] init];
+        [[NSNotificationCenter defaultCenter] postNotificationName:SFUserAccountManagerDidFinishUserInitNotification
```

Comment:
This also causes a deadlock since we are still within the dispatch_once. I tried locally and it is fixed by moving it outside of the dispatch_once and using 

[NSNotificationCenter postNotificationOnceWithName:SFUserAccountManagerDidFinishUserInitNotification object:nil userInfo:nil];